src/Symfony/Installer/NewCommand.php

```diff
@@ -210,9 +227,8 @@ private function download()
         $this->output->writeln("\n Downloading Symfony...");
 
         // decide which is the best compressed version to download
-        $distill = new Distill();
         $baseName = 'http://symfony.com/download?v=Symfony_Standard_Vendors_'.$this->version;
-        $symfonyArchiveFile = $distill
+        $symfonyArchiveFile = $this->distill
```

Comment:
given that ``Distill`` is a stateful object, using it as a reusable dependency looks wrong